Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/03-SyncUpForm/TestingSyncUpForm-02-code-0005.swift

```diff
@@ -28,7 +28,7 @@ struct SyncUpForm {
     Reduce { state, action in
       switch action {
       case .addAttendeeButtonTapped:
-        let attendee = Attendee(id: uuid())
+        let attendee = Attendee(id: Attendee.ID(uuid()))
```

Comment:
The digression about `Tagged` was only optional and we had a note that the rest of the tutorial does not assume we are using `Tagged`.

This has actually caused a lot of confusion with people following the tutorial, so we recently removed the `Tagged` digression entirely. As such would you mind reverting this and the change below?

Otherwise it looks good! Thanks!

Reply:
Thanks for your comment.
I will now only make the correction to the filename.

Reply:
@mbrandonw 

I did the Syncups tutorial to the end, with a few confusing points and typos other than `Tagged`. Then I saw `Coming soon!` in the [Adding a speech recognizer](https://pointfreeco.github.io/swift-composable-architecture/main/tutorials/composablearchitecture/implementingspeechrecognizer#Asking-for-speech-recognition-permission) section at the end. So If you're modifying the tutorial now and this PR gets in the way, it's fine to close it.

Reply:
Oh sorry, no I just lost track of this. These changes can definitely be merged in, and we do have more fixes and more content coming sometime in the future.